lib/kits/paged-publication/hotspots.ts

```diff
@@ -45,6 +45,23 @@ function getPosition(pages: Page[], ratio: number, hotspot: V2Hotspot) {
     };
 }
 
+function getHostname(link: string) {
```

Comment:
I think let's remove the link icon and hostname and instead just write in the top/right corner: "Read more", which is localized and which you can override

Reply:
Here's how it will look like:
![image](https://github.com/tjek/tjek-js-sdk/assets/32891190/34d67937-d674-41e6-8368-db2b94fb4883)